.dockerignore

```diff
@@ -5,3 +5,7 @@
 .gitattributes
 .dockerignore
 Dockerfile
+bin/run-in-docker.sh
+bin/run-tests-in-docker.sh
+bin/run-tests.sh
+test/
```

Comment:
This makes the Docker image just a tad lighter as we don't unnecessary include the test files :)